deps/chakrashim/src/jsrtisolateshim.cc

```diff
@@ -844,7 +844,7 @@ bool IsolateShim::RunSingleStepOfReverseMoveLoop(v8::Isolate* isolate,
     int64_t snapEventTime = -1;
     int64_t snapEventEndTime = -1;
     int64_t origNETime = *nextEventTime;
-    JsTTDMoveMode _moveMode = (JsTTDMoveMode)(*moveMode);
+    JsTTDMoveMode _moveMode = (JsTTDMoveMode)(*moveMode);  // TODO JsTTDMoveMode is no longer 64 bit wide, fix use of moveMode parameter
```

Comment:
I did run cpplint actually, and I missed this one because jsrtisolateshim.cc has a lot of warnings.  The other files are okay, two have preexisting warnings about not using int16/int64/etc types instead of short, which I will not touch.